topics/first-tech-challenge/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: ftc, first-ftc, ftc-robotics, ftc-sdk
```

Comment:
It looks like `ftc` is more widely used than `first-tech-challenge`; thoughts on making it the primary name?